scripts/lib/setup/installers.sh

```diff
@@ -374,6 +426,7 @@ function use_android_sdk() {
     local ANDROID_PLATFORM_VERSION=$(toolversion ANDROID_PLATFORM_VERSION)
     touch ~/.android/repositories.cfg
     echo y | sdkmanager "platform-tools" "build-tools;$ANDROID_BUILD_TOOLS_VERSION" "platforms;$ANDROID_PLATFORM_VERSION"
+    sdkmanager --licenses
```

Comment:
You should do this before you start installing with `sdkmanager`:
```
yes | sdkmanager --licenses
```

Reply:
Or do you want to let the user accept it manually?

Reply:
Well, the thing is you already accept them for the user in line 428 anyway.

Reply:
Do we want to automatically accept licenses for the user without letting him know?

Reply:
Ethically? Probably not.
Practically? Yes
I mean, who really reads those? Exactly and precisely no one.

Reply:
OK, I'll add it then.